manifests/a/ArduinoSA/IDE/stable/2.0.2/ArduinoSA.IDE.stable.locale.en-US.yaml

```
@@ -0,0 +1,32 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: ArduinoSA.IDE.stable
+PackageVersion: 2.0.2
+PackageLocale: en-US
+Publisher: Arduino SA
+PublisherUrl: https://www.arduino.cc
+PublisherSupportUrl: https://support.arduino.cc
+PrivacyUrl: https://www.arduino.cc/en/privacy-policy
+Author: Arduino SA
+PackageName: Arduino IDE
+PackageUrl: https://www.arduino.cc/en/software
+License: AGPLv3
+LicenseUrl: https://github.com/arduino/arduino-ide/blob/main/LICENSE.txt
+Copyright: Copyright © 2022 Arduino SA
+ShortDescription: Arduino IDE (stable version)

```

```suggestion
ShortDescription: Open-source electronic prototyping platform enabling users to create interactive electronic objects.
```
ShortDescription should be meaningful